blog/content/post/2024-11-04-qubes-os-summit-post.md

```diff
@@ -0,0 +1,226 @@
+---
+title: 'Qubes Os Summit 2024'
+abstract: 'The sixth edition of the Qubes OS Summit, organized by Invisible 
+Things Lab and 3mdeb, took place at the end of September'
+cover: /covers/image-file.png
+author: marta.witkowska
```

Comment:
@MartaWitt this is based on my notes, so I would appreciate correct co-authoring:
https://github.com/3mdeb/news-and-ideas?tab=readme-ov-file#single-or-multiple-authors

Reply:
@MartaWitt, I take this back after reading it carefully. The post has little to do with my notes so that you can remove me from the authors. Sorry for the noise.

Reply:
Removed.